sdk/client/src/listeners/first-load-listeners.tsx

```diff
@@ -258,25 +276,31 @@ export class FirstLoadListeners {
 				})
 
 			if (sThis.enableRecordingNetworkContents) {
-				resources = matchPerformanceTimingsWithRequestResponsePair(
-					resources,
+				httpResources = matchPerformanceTimingsWithRequestResponsePair(
+					httpResources,
 					sThis.xhrNetworkContents,
 					'xmlhttprequest',
 				)
-				resources = matchPerformanceTimingsWithRequestResponsePair(
-					resources,
+				httpResources = matchPerformanceTimingsWithRequestResponsePair(
+					httpResources,
 					sThis.fetchNetworkContents,
 					'fetch',
 				)
 			}
 		}
-		return resources
+
+		if (!sThis.disableRecordingWebSocketContents) {
+			webSocketResources = sThis.webSocketNetworkContents
+		}
+
+		return [...httpResources, ...webSocketResources]
```

Comment:
Assuming these do not need to be ordered by timestamp since we do this on the frontend, but should we do that here?